### ISSUE TITLE
Request to change jog wheel behaviour.

### ISSUE THREAD

**Issue by mikobuntu** [nikmartin/hercules-mixxx-mapping#1]
Hi Nik, thanks for your great script, I am having fun using it with Mixxx on my Ubuntu box.
I have found one behaviour which I think needs slightly adapted ( if possible? ). 
Right now as you know when using the jog wheel during playback we get the default scratch when pressed and minor adjustment for beat matching when not pressed which is perfect, my concern is that I sometimes scratch on a deck without actually having play engaged, which causes the song browser scrolling to engage sometimes during press and release of the jog wheels as the momentum carries the actual jog wheel, causing the browser choice to move from my last selection.
So basically I think this could be fixed by imposing the user to use a hot key in conjunction with the jog wheel ( during non playback of said track ) to give focus to the song browser.
If you wanted to keep all keystrokes to the controller itself, then perhaps the "Files" button along with jog wheel could be used? I also see this freeing up the jog wheel to do more coarse adjustment to the audio file's position. thanks :)
